bindings/c/generate_asm.py

```diff
@@ -74,10 +74,41 @@ def write_unix_asm(asmfile, functions, prefix):
     for f in functions:
         asmfile.write("\n.globl %s%s\n" % (prefix, f))
         asmfile.write("%s%s:\n" % (prefix, f))
+
+        # These assembly implementations of versioned fdb c api functions must have the following properties.
+        #
+        # 1. Don't require dynamic relocation.
+        #
+        # 2. Perform a tail-call to the function pointer that works for a
```

Comment:
out of curiosity, how did you find the bug? what tools did you use?

Reply:
It simply didn't compile with `clang` 😅